frontend/src/index.html

```diff
@@ -8,7 +8,7 @@
 ---
 <div class="md:flex">
   <!-- Body left -->
-  <div class="md:w-full">
+  <div class="md:w-full md:flex-4">
```

Comment:
- Why do we need `w-full` here?
- Instead of `flex-` I think we can directly use [grow-](https://tailwindcss.com/docs/flex-grow#arbitrary-values).

Reply:
`w-full` removed